### ISSUE TITLE
Link to source code for demos are broken

### ISSUE THREAD

**Issue by s-edlund** [swimlane/ngx-datatable#1740]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
Clicking any link to source code gives a 404

**Expected behavior**
Links should work

**Reproduction of the problem**
Go to http://swimlane.github.io/ngx-datatable/ and click `Source`. It gives a 404

**Comment by marjan-georgiev** [comment]
This is fixed. We released a new version (16.0.0) and updated the demo pages today. Would encourage you to upgrade.

**Comment by adrynov** [comment]
Source code for demos link is broken

https://github.com/swimlane/ngx-datatable/tree/master/demo
